.github/workflows/test.yml

```diff
@@ -37,8 +37,8 @@ jobs:
           go version
           export GOMAXPROCS=2
           args=("-p" "2" "-race")
-          # Run with less concurrency on Windows to minimize flakiness.
-          if [[ "${{ matrix.platform }}" == windows* ]]; then
+          # Run with less concurrency on Windows/MacOS to minimize flakiness.
```

Comment:
The issue is that tests are flaky on all platforms, primarily GRPC-TLS related https://github.com/grafana/xk6-grpc/issues/39. Which ones do you constantly see?

Reply:
Yeah, there's nothing new, as I think all (or most) of them are documented at https://github.com/grafana/k6/issues/2144.

It's just that it seems that Windows and MacOS runners are way more likely to fail (it's probably a matter of their performance), so I decided to add MacOS into that exception we already had for Windows, to avoid adding more flakiness.